src/utils/src/log.js

```diff
@@ -18,13 +22,9 @@ Log.methodFactory = function methodFactory(methodName, logLevel, loggerName) {
     }
   };
 };
+log.setLevel(log.getLevel());
```

Comment:
@thomasconner this looks odd. This should be a No-Op if getters and setters do what they're supposed to :)